pyop2/base.py

```diff
@@ -4272,39 +4149,18 @@ def iteration_region(self):
         interior facets."""
         return self._iteration_region
 
-    def _build_itspace(self, iterset):
-        """Creates an class:`IterationSpace` for the :class:`ParLoop` from the
-        given iteration set.
-
-        Also checks that the iteration set of the :class:`ParLoop` matches the
-        iteration set of all its arguments. A :class:`MapValueError` is raised
-        if this condition is not met.
-
-        Also determines the size of the local iteration space and checks all
-        arguments using an :class:`IterationIndex` for consistency.
 
-        :return: class:`IterationSpace` for this :class:`ParLoop`"""
-        return build_itspace(self.args, iterset)
-
-
-def build_itspace(args, iterset):
-    """Creates an class:`IterationSpace` for the :class:`ParLoop` from the
-    given iteration set.
-
-    Also checks that the iteration set of the :class:`ParLoop` matches the
+def check_iterset(args, iterset):
+    """Checks that the iteration set of the :class:`ParLoop` matches the
     iteration set of all its arguments. A :class:`MapValueError` is raised
     if this condition is not met.
-
```

Comment:
This looks like the accidental removal of a healthy blank line (?)

Reply:
Think so. Added back.